src/olympia/editors/tests/test_models.py

```diff
@@ -235,17 +235,10 @@ def test_lite_review_addons_also_shows_up(self):
                           listed=self.listed)
         create_addon_file('Lite', '0.1',
                           amo.STATUS_LITE_AND_NOMINATED,
-                          amo.STATUS_UNREVIEWED, listed=self.listed)
+                          amo.STATUS_LITE, listed=self.listed)
         assert sorted(q.addon_name for q in self.Queue.objects.all()) == (
             ['Full', 'Lite'])
 
-    def test_any_nominated_file_shows_up(self):
-        create_addon_file('Null', '0.1',
-                          amo.STATUS_NOMINATED, amo.STATUS_NULL,
-                          listed=self.listed)
-        assert sorted(q.addon_name for q in self.Queue.objects.all()) == [
-            'Null']
-
```

Comment:
This test was checking for a file status that should never [happen](https://github.com/mozilla/addons-server/blob/master/src/olympia/constants/base.py#L48) so it failed.